src/controllers/serverStatus.controller.ts

```diff
@@ -0,0 +1,9 @@
+import { type Request, type Response } from 'express';
```

Comment:
import { Request, Response } from 'express';

Reply:
Indeed, type here is pretty much useless and removing it would improve cleanliness of my code. Thank you for advice, consider it done!